src/routes/_store/computations/navComputations.js

```diff
@@ -24,6 +24,13 @@ export function navComputations (store) {
           svg: '#fa-globe',
           label: 'Federated'
         }
+      } else if (pinnedPage === '/conversations') {
+        pinnedPageObject = {
+          name: 'conversations',
+          href: '/conversations',
+          svg: '#fa-envelope',
+          label: 'Conversations'
+        }
```

Comment:
In general, I prefer to match Mastodon's UI and URL conventions whenever possible. It seems that the Mastodon web interface uses `/direct` for the URL and "Direct messages" for the user-facing text. So I'd prefer to stick to that if possible.